tslint.json

```diff
@@ -4,6 +4,8 @@
   },
   "extends": "tslint:all",
   "rules": {
+    "linebreak-style": false,
```

Comment:
This was probably not intended to be commited. Please revert this change.

Reply:
Agh..